src/commands/deploy/deploy.js

```diff
@@ -381,6 +386,40 @@ const handleBuild = async ({ cachedConfig, options }) => {
   return { newConfig, configMutations }
 }
 
+/**
+ *
+ * @param {object} options Bundling options
+ * @returns
+ */
+const bundleEdgeFunctions = async (options) => {
+  const { runCoreSteps } = await runCoreStepPromise
+  const statusCb = options.silent ? () => {} : deployProgressCb()
+
+  statusCb({
+    type: 'edge-functions-bundling',
+    msg: 'Bundling edge functions...\n',
+    phase: 'start',
+  })
+
+  const { severityCode, success } = await runCoreSteps(['edge_functions_bundling'], { ...options, buffer: true })
```

Comment:
@jackiewmacharia I changed this to use object spread instead of `Object.assign` to avoid mutating `options`. Let me know if that's okay with you.

Reply:
Looks good! I'll merge this and the subsequent release PR.